src/rdf.test.ts

```diff
@@ -212,7 +214,7 @@ describe("fromRdfJsDataset", () => {
     expect(fromRdfJsDataset(rdfJsDataset)).toStrictEqual({
       type: "Dataset",
       graphs: {
-        default: {
+        default: expect.objectContaining({
```

Comment:
We no longer do exact matches on the internal representation, because of the unstable blank nodes identifiers.